pkgs/development/python-modules/bpython/default.nix

```diff
@@ -24,7 +26,7 @@ buildPythonPackage rec {
     which = "${which}/bin/which";
   })];
 
-  propagatedBuildInputs = [ curtsies greenlet pygments requests urwid ];
+  propagatedBuildInputs = [ curtsies greenlet pygments requests urwid pyxdg ];
```

Comment:
These could be sorted alphabetically on separate lines.

Reply:
Done